service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/DraftOrdersController.java

```diff
@@ -226,4 +211,15 @@ private Document getDocument(String authorization, String userId, Map<String, Ob
 
         return uploadDocumentService.uploadPDF(userId, authorization, document.getBytes(), docTitle);
     }
+
+    private List<String> validate(CaseData caseData) {
```

Comment:
this probably does not belong in the controller. It also means you have to test in integration tests which isn't great. I think I would rather some sort of error message validation service. We can then extensively unit test and then cut down on the integration tests. What are your thoughts?

Reply:
Absolutely agree it does not belong in here. Went with this option as its a minimal update and tried to reduce the PR size. Also, I think we will still need the integration test for the new scenario?

Reply:
hmm, so yes, a integration test is probably required. But with extensinve units test, the four we currently have would not be required

Reply:
updated